hotsos/core/plugintools.py

```diff
@@ -324,6 +324,19 @@ def all(self):
         return {HotSOSConfig.plugin_name: parts}
 
 
+def summary_function(name, index=0):
```

Comment:
can we name this: def summary_entry(name, index=0)

Reply:
done